ChildProject/annotations.py

```diff
@@ -458,7 +459,6 @@ def _check_for_outdated_merged_sets(self, sets: set = None):
                             warnings.append("set {} is outdated because the {} set it is merged from was modified. Consider updating or rerunning the creation of the {} set.".format(i,j,i))
                         
         return warnings
-        
     def _import_annotation(
         self, import_function: Callable[[str], pd.DataFrame],
         params: dict,
```

Comment:
overwrite_existing - great idea. Can we change the error from: "to reimport it, use the overwrite_existing flag"; to "to reimport it, use overwrite_existing = TRUE" or something more precisely actionable like that?

Reply:
actually, I'm seeing that the definitions are also a bit cryptic... Perhaps we can discuss how to rephrase them or provide pointers to necessary definitions?

Reply:
ha, I just realized that this is an old function! Moving on...